bugwarrior/config/schema.py

```diff
@@ -172,104 +197,80 @@ def compute_data(cls, values):
     merge_annotations: bool = True
     merge_tags: bool = True
     replace_tags: bool = False
-    static_tags: ConfigList = ConfigList([])
-    static_fields: ConfigList = ConfigList(['priority'])
+    static_tags: ConfigList = []
+    static_fields: ConfigList = ["priority"]
 
-    log_level: typing.Literal[
-        ('DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL', 'DISABLED')
-    ] = 'INFO'
+    log_level: Literal["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL", "DISABLED"] = (
+        "INFO"
+    )
     log_file: typing.Optional[LoggingPath] = None
 
 
-class Hooks(pydantic.v1.BaseModel):
-    pre_import: ConfigList = ConfigList([])
+class Hooks(BaseConfig):
+    pre_import: ConfigList = []
 
 
-class Notifications(pydantic.v1.BaseModel):
+class Notifications(BaseConfig):
     notifications: bool = False
-    # Although upstream supports it, pydantic has problems with Literal[None].
-    backend: typing.Optional[
-        typing.Literal[('gobject', 'growlnotify', 'applescript')]
-    ] = None
+    backend: typing.Optional[Literal["gobject", "growlnotify", "applescript"]] = None
     finished_querying_sticky: bool = True
     task_crud_sticky: bool = True
     only_on_new_tasks: bool = False
 
 
-class SchemaBase(pydantic.v1.BaseSettings):
-    class Config(PydanticConfig):
-        # Allow extra top-level sections so all targets don't have to be selected.
-        extra = 'ignore'
+class SchemaBase(BaseConfig):
+    model_config = ConfigDict(extra="ignore")
```

Comment:
Add back comment so we remember why we're doing this.

```suggestion
    # Allow extra top-level sections so all targets don't have to be selected.
    model_config = ConfigDict(extra="ignore")
```